components/salesforce_rest_api/salesforce_rest_api.app.mjs

```diff
@@ -201,23 +197,14 @@ export default {
         sobject.name.includes("History")
       );
     },
-    listAllowedSObjectTypes(eventType) {
-      const verbose = true;
-      return SalesforceClient.getAllowedSObjects(eventType, verbose);
-    },
     async createWebhook(endpointUrl, sObjectType, event, secretToken, opts) {
-      const {
-        fieldsToCheck,
-        fieldsToCheckMode,
-      } = opts;
       const client = this._getSalesforceClient();
       const webhookOpts = {
         endpointUrl,
         sObjectType,
         event,
         secretToken,
-        fieldsToCheck,
-        fieldsToCheckMode,
+        ...opts,
       };
```

Comment:
We should skip the SObject validation! 😄 
```suggestion
        skipValidation: true,
      };
```

Reply:
Nevermind, already done here:
https://github.com/PipedreamHQ/pipedream/pull/4196/files/d7402313a6dba44c3f55d0242d94738044c3e430#